pypeit/wavecalib.py

```diff
@@ -628,18 +635,27 @@ def build_wv_calib(self, arccen, method, skip_QA=False,
         elif method == 'reidentify':
             # Now preferred
             # Slit positions
-            arcfitter = autoid.ArchiveReid(arccen, self.lamps, self.par,
-                                           ech_fixed_format=self.spectrograph.ech_fixed_format, ok_mask=ok_mask_idx,
-                                           measured_fwhms=measured_fwhms,
-                                           orders=self.orders,
-                                           nonlinear_counts=self.nonlinear_counts)
+            arcfitter = autoid.ArchiveReid(
+                arccen, self.lamps, self.par,
+                ech_fixed_format=self.spectrograph.ech_fixed_format, 
+                ok_mask=ok_mask_idx,
+                measured_fwhms=self.measured_fwhms,
+                orders=self.orders,
+                nonlinear_counts=self.nonlinear_counts)
             patt_dict, final_fit = arcfitter.get_results()
+
+            # Save arxiv for redo later?
```

Comment:
Should these two comments replace `Save` with `Load`?

Reply:
I replaced with "grab" and "hold"